tg/msg.py

```diff
@@ -125,3 +125,35 @@ def is_text(self):
     def is_downloaded(self):
         doc = self.get_doc(self.msg)
         return doc["local"]["is_downloading_completed"]
+
+    @property
+    def is_listened(self):
+        if self.type != "voice":
+            return None
+        if self.msg["content"]["is_listened"]:
+            return "yes"
+        return "no"
+
+    @is_listened.setter
+    def is_listened(self, value):
```

Comment:
plz, add argument and return type annotations for new methods

Reply:
To avoid such comments, let's enable mypy. Until mypy is in CI, nobody, except you :) will annotate them.